### PR TITLE
Add dir structure in tree format

### PR DESCRIPTION
**Pull request by akionsight**
Added dir structure in tree format

### CODE REVIEW

**Comment by akionsight** [comment]
this is added to the CONTRIBUTE.md for easier understanding of the code base

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -33,4 +33,157 @@ def get_context(self, request: WSGIRequest) -> dict:
 like this
 please add this in the context of the view because this loads all the pfp and permissions
 
+## directory structure
+The following is the tree representation of the code jam website
+```
+.
+├── CONTRIBUTE.md
```
Comment (Shivansh-007) [inline CONTRIBUTE.md:40]: Why not link a file, this is probably not looking good in the Contributing guide.

**Comment by Shubhaankar-sharma** [comment]
hmm yes this clutters the contribute.md rather create another file called structure.md and copy and paste @akionsight

**Comment by akionsight** [comment]
Done. I made a separate file called STRUCTURE.md and added it's reference to the README.md too

File: CONTRIBUTE.md (hunk 1 above)
Comment (akionsight) [reply CONTRIBUTE.md]: Done. See below for more info

**Pull request merged**